webapp/channels/src/components/post/post_options.tsx

```diff
@@ -193,28 +201,31 @@ const PostOptions = (props: Props): JSX.Element => {
                 if (item.component) {
                     const Component = item.component as any;
                     return (
-                        <Component
-                            post={props.post}
-                            key={item.id}
-                        />
+                        <li key={item.id}>
+                            <Component
```

Comment:
@jespino FYI on this since it's going to change the HTML around the post menu for the AI plugin, although I don't think it's likely to cause any actual issues